CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * [#4249](https://github.com/bbatsov/rubocop/issues/4249): Handle multiple assignment in `Rails/RelativeDateConstant`. ([@bbatsov][])
 * [#4250](https://github.com/bbatsov/rubocop/issues/4250): Improve a bit the Ruby code detection config. ([@bbatsov][])
 * [#4283](https://github.com/bbatsov/rubocop/issues/4283): Fix `Style/EmptyCaseCondition` autocorrect bug - when first `when` branch includes comma-delimited alternatives. ([@ilansh][])
+* [#3400](https://github.com/bbatsov/rubocop/issues/3400): Fix `Lint/Debugger` - cancel autocorrection for this cop due to buggy implementation. ([@ilansh][])
```

Comment:
This should be in "Changes" and should simply say something like "Remove auto-correct support from `Lint/Debugger`".

Same goes for the commit message.